bin/drush-install.sh

```diff
@@ -35,6 +35,31 @@ org.civicrm.contactsummary,\
 org.civicrm.bootstrapcivicrm,\
 org.civicrm.bootstrapcivihr
 
+## List of Default Enabled Cron Jobs
+ENABLED_JOBS=( "CiviCRM Update Check" \
+"Clean-up Temporary Data and Files" \
+"Clone Documents" \
+"Disable expired relationships" \
+"Length of service updater" \
+"Mail Reports" \
+"Process Inbound Emails" \
+"Send Scheduled Mailings" \
+"Send Scheduled Reminders" \
+"Tasks and Assignments Daily Reminder" \
+"Documents Notification" \
```

Comment:
I already left a comment about all of this , anything that could be done in an extension should be done in an extension and since we have hrcore extension now we can put these things there .. but you presented another important note here which is civihr extensions cron jobs .. I think you are right .. so only core cron jobs should be disabled/enabled in hrcore and the rest should be done in their related extensions .